config/webpack/configPlugins.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-param-reassign */
+const webpack = require('webpack');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
+
+const webpackConfigPlugins = (config, { dev }) => {
+  const plugins = [
+    new webpack.IgnorePlugin(
+      /pages.*\/test.*/,
+    ),
+  ];
+
+  if (process.env.NODE_ENV === 'analyse') {
```

Comment:
do does that mean that in order to use the analyse mode, people will need to set the `NODE_ENV` variable to `analyse`? Is there any action required by developers?

Reply:
Technically yes, but we have a shorthand for it, it is `npm run analyze`, it is in the README. Mind you, `npm run analyze` only outputs a HTML file, other than that it behaves exactly like building the platform normally, so it's not like an additional feature that is enabled.

Reply:
I can't see it in the readme.